app/js/lib/foxbox/foxbox.js

```diff
@@ -355,7 +355,7 @@ export default class Foxbox extends Service {
     return this[p.net].fetchJSON(`${this[p.net].origin}/services/${id}/state`)
       .then(res => {
         if (!res) {
-          throw new Error(`The action couldn't be performed.`);
+          throw new Error('The action couldn\'t be performed.');
```

Comment:
Wait, eslint don't support back tick as a regular string delimiter? That's a major annoyance to force using single quote. Can't we ignore these warning for now and maybe later write our own plugin for that?